src/NewRecipe.css

```diff
@@ -0,0 +1,65 @@
+.New {
+  margin-bottom: 50px auto;
+  margin-top: 0px auto;
```

Comment:
If you have multiple margins/paddings, you can add them in a single line like so: 
p {
  margin: 25px 50px 75px 100px;
}
https://www.w3schools.com/css/css_margin.asp